pkg/controller/clone_controller.go

```diff
@@ -79,8 +79,6 @@ func (p *CloningProtectionController) Run(ctx context.Context, threadiness int)
 		}, time.Second, ctx.Done())
 	}
 
-	go p.claimInformer.Run(ctx.Done())
```

Comment:
This is not related to this change, but I think it's better that we move adding the `claimHandler` to the `New` function so that this `Run` function does not depend on ordering of the other `Run` functions.